### DISCUSSION TITLE
Welcome to FreeStyle-Libre-3-patch Discussions!

### DISCUSSION THREAD

**Discussion by maheini** [maheini/FreeStyle-Libre-3-patch#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by gretajohanna** [comment]
Hi there!
Libre-3-Patch App does Not work with iPhone, or? I did the download but unfortunately I cannot start it….

**Comment by maheini** [comment]
No, there's no chance getting this patch working with iOS. Android only.

**Reply by gretajohanna** [reply]
What a pity! Thanks for your quick answer!

**Comment by NotYourMothersITGuy** [comment]
I have been successfully using Juggluco with my US Libre 3 sensor for over a week. I haven't had any issues and being able to see my glucose on the top of my screen at all times has been a game changer. 

This morning I decided I wanted to get it working on my Pixel Watch. I downloaded the GlucoseDataHandler app from the Playstore to my phone and watch, opened the Juggluco app and enabled Glucodata Broadcast. This took all of about 3 minutes, and now I can see my glucose level on my watch face.
![image](https://github.com/maheini/FreeStyle-Libre-3-patch/assets/109396226/6b0fae9c-7fee-4787-b5ac-92248fc8b044)

edit: I had to change sensors for the first time since using Juggluco. I scanned the new sensor using the original Libre 3 app. After the 60 minutes, I had to disable/enable Glucodata Broadcast and within a few minutes, everything was working again! The last couple weeks, my blood sugar has been better than ever because of the constant visibility. I'm so happy with everything now. When I found I could only get the Libre CGM, I was a little disappointed because I thought it meant I would never be able to integrate with Home Assistant, view on my smartwatch, etc.
